### DISCUSSION TITLE
Project manager not detecting git workspaces in wsl

### DISCUSSION THREAD

**Discussion by gykovacs** [alefragnani/vscode-project-manager#681] [Q&A] [answered]
Hi! I run into the problem what despite any baseFolder configuration, project manager is not able to detect the projects on WSL. Note that this setup is a vscode server in WSL and a frontend in Windows.

![image](https://user-images.githubusercontent.com/1881098/229342337-76ea6bee-63a9-44b3-9b53-152c19554b78.png)

**Comment by alefragnani** [comment, marked as answer]
Hi @gykovacs ,

The `baseFolders` settings does not support remotes. If you need remote locations being recognized, you must install the extension on that remote. There is a documentation in README that points out what you need to do.

On the other hand, there is an open issue (#487) to add support for this. If installing the extension on remote doesn't fit your need, I suggest you to upvote/subscribe on that issue, to be notified of updated.

Hope this helps
